### PR TITLE
Removed Base64 encoded fonts

### PR DESCRIPTION
**Pull request by Nooshu**
Removed the Base64 encoded fonts to remove this data from the browsers critical path. This has been replaced by standard `@font-face` declarations. More information of this change can be seen in [PR 341](https://github.com/alphagov/govuk_template/pull/341) from govuk_template.

I've tested these changes in the example application and they work as expected.

### CODE REVIEW

File: lib/assets/stylesheets/utilities/_fonts.scss [hunk 1]
```diff
@@ -1,29 +1,47 @@
-/* Warning: this file is automatically generated by `gds/fonts` - do not edit by hand */
```
Comment (tijmenb) [inline lib/assets/stylesheets/utilities/_fonts.scss]: Any idea why this is duplicated from https://github.com/alphagov/tech-docs-gem/blob/master/lib/assets/stylesheets/_fonts.scss?

Comment (Nooshu) [reply lib/assets/stylesheets/utilities/_fonts.scss]: Utilities is the scss file that changes the fonts, the ` /lib/assets/stylesheets/_fonts.scss` looks redundant to me. I will try removing the redundant one and see if anything breaks.

Comment (Nooshu) [reply lib/assets/stylesheets/utilities/_fonts.scss]: Confirmed that the extra font file is redundant. Removed and squashed the commits.

**Comment by Nooshu** [comment]
Closing this as this is covered by [this PR](#34).

**Pull request closed without merging**